frontend/public/kinds.ts

```diff
@@ -51,9 +52,21 @@ export const connectToPlural: ConnectToPlural = connect(
   ) => {
     const plural = props.plural || _.get(props, 'match.params.plural');
 
-    const kindObj = isGroupVersionKind(plural)
-      ? state.k8s.getIn(['RESOURCES', 'models']).get(plural)
-      : allModels().find((model) => model.plural === plural);
+    const groupVersionKind = getGroupVersionKind(plural);
```

Comment:
/cc @vojtechszocs @spadgett 

@christianvogt and I discussed this logic change to handle grabbing from local k8s models rather than defaulting to an API model so we could use the dev/tech preview badge as a model-oriented property.